src/ui/roles/list/RoleListMenuActions.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { DropdownKebab, MenuItem } from 'patternfly-react';
+// import { LinkMenuItem } from 'frontend-common-components';
+// import { ROUTE_GROUP_EDIT, ROUTE_GROUP_DETAIL } from 'app-init/router';
+
+class RoleListMenuActions extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(handler) {
+    return (ev) => {
+      ev.preventDefault();
+      if (handler) { handler(this.props); }
+    };
+  }
+
+  render() {
+    const { onClickDelete } = this.props;
+
+    // const editLabel = <FormattedMessage id="app.edit" />;
+    // const editDetails = <FormattedMessage id="app.details" />;
+    return (
+      <DropdownKebab pullRight id={`${this.props.code}-actions`}>
+        {/* <LinkMenuItem
```

Comment:
Can you please create the route in "app-init/router", and activate all the links? This way, we'll have all the links in place, pointing to a 404 page, and when we create the related page we just have to update App.js.